packages/block-library/src/navigation/edit/placeholder/placeholder-preview.js

```diff
@@ -9,12 +9,11 @@ import classnames from 'classnames';
 import { Icon, navigation } from '@wordpress/icons';
 import { __ } from '@wordpress/i18n';
 
-const PlaceholderPreview = ( { isLoading } ) => {
```

Comment:
Loading variant seemed confusing. Not sure why it was necessary.

Reply:
> Not sure why it was necessary.

I should have qualified that statement. I understood why it existed but not it's no longer required due to refactoring.

> Maybe it can be deleted, but I haven't checked to see where it's used.

It _was_ also used as the `placeholder` prop arg for the `<InnerBlocks>` component which would show when the inner blocks were _empty_ but present and _not selected_.

I'll check whether it can be ditched. It would certain make things less confusing 😄